### PR TITLE
Add unit test config to settings.gradle for Java

### PR DESCRIPTION
**Pull request by sciencewhiz**
Adapted from https://www.chiefdelphi.com/t/unit-testing-java-io-ioexception-wpihaljni/372288/19

### CODE REVIEW

File: vscode-wpilib/resources/gradle/java/.vscode/settings.json [hunk 1]
```diff
@@ -13,5 +13,16 @@
...
     "**/*~": true
-  }
+  },
+  "java.test.config": [
+    {
+        "name": "WPIlibUnitTests",
+        "workingDirectory": "${workspaceFolder}/build/jni/debug",
+        "vmargs": [ "-Djava.library.path=${workspaceFolder}/build/jni/debug" ],
+        "env": { "LD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug" ,
+          "DYLD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug" }
```
Comment (PeterJohnson) [inline vscode-wpilib/resources/gradle/java/.vscode/settings.json:23]: Formatting
```suggestion
        "env": {
          "LD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug",
          "DYLD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug"
        }
```

File: vscode-wpilib/resources/gradle/javadt/.vscode/settings.json [hunk 2]
```diff
@@ -13,5 +13,16 @@
...
     "**/*~": true
-  }
+  },
+  "java.test.config": [
+    {
+        "name": "WPIlibUnitTests",
+        "workingDirectory": "${workspaceFolder}/build/jni/debug",
+        "vmargs": [ "-Djava.library.path=${workspaceFolder}/build/jni/debug" ],
+        "env": { "LD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug" ,
+          "DYLD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug" }
```
Comment (PeterJohnson) [inline vscode-wpilib/resources/gradle/javadt/.vscode/settings.json:23]:
```suggestion
        "env": {
          "LD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug",
          "DYLD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug"
        }
```

File: vscode-wpilib/resources/gradle/javaromi/.vscode/settings.json [hunk 3]
```diff
@@ -14,5 +14,16 @@
...
   },
-  "wpilib.skipSelectSimulateExtension": true
+  "wpilib.skipSelectSimulateExtension": true,
+  "java.test.config": [
+    {
+        "name": "WPIlibUnitTests",
+        "workingDirectory": "${workspaceFolder}/build/jni/debug",
+        "vmargs": [ "-Djava.library.path=${workspaceFolder}/build/jni/debug" ],
+        "env": { "LD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug" ,
+          "DYLD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug" }
```
Comment (PeterJohnson) [inline vscode-wpilib/resources/gradle/javaromi/.vscode/settings.json:24]:
```suggestion
        "env": {
          "LD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug",
          "DYLD_LIBRARY_PATH": "${workspaceFolder}/build/jni/debug"
        }
```

**Review by PeterJohnson — CHANGES_REQUESTED** [review]
Really the whole file needs a bit of reformatting, but fixed the additions.

**Comment by sciencewhiz** [comment]
This may also have to changed based on the resolution to #475

**Comment by PeterJohnson** [comment]
Now that 480 has been merged, this just needs updating to point to release.

**Review by ThadHouse — APPROVED** [review]

**Pull request merged**